package.json

```diff
@@ -742,15 +742,15 @@
 		"vsce": "^1.37.5"
 	},
 	"dependencies": {
-		"antlr4ts": "^0.4.0-alpha.4",
+		"antlr4ts": "^0.4.0-burt.2",
```

Comment:
?

Reply:
Don't know why it picked that up. Please revert or change to 0.4.1 alpha

![image](https://user-images.githubusercontent.com/6913354/38157944-0082abd6-3441-11e8-8ea5-592ca2e66dc5.png)